integration_tests/seeds/expected/base_average_metric__expected.csv

```diff
@@ -1,303 +1,5 @@
-2022-03-27,TRUE,26.000000
-2022-02-04,FALSE,17.250000
-2022-03-13,TRUE,27.250000
-2022-05-28,FALSE,19.600000
-2022-01-12,FALSE,26.500000
-2022-02-02,FALSE,35.250000
-2022-02-02,TRUE,25.200000
-2022-02-09,FALSE,25.333333
-2022-02-13,FALSE,31.000000
-2022-02-21,FALSE,1.000000
-2022-02-27,FALSE,26.000000
-2022-03-05,FALSE,29.333333
-2022-03-06,TRUE,16.000000
-2022-03-14,TRUE,15.000000
-2022-03-15,TRUE,41.333333
-2022-03-22,TRUE,30.000000
-2022-03-26,FALSE,25.000000
-2022-03-26,TRUE,19.000000
-2022-03-28,FALSE,22.250000
-2022-04-06,FALSE,26.000000
-2022-04-08,FALSE,22.666667
-2022-04-17,TRUE,26.666667
-2022-04-18,TRUE,25.000000
-2022-04-22,FALSE,34.500000
-2022-04-23,TRUE,23.166667
-2022-04-29,FALSE,40.250000
-2022-04-30,FALSE,10.000000
-2022-05-01,FALSE,11.000000
-2022-05-04,TRUE,31.200000
-2022-05-09,FALSE,21.400000
-2022-05-09,TRUE,26.000000
-2022-05-11,FALSE,27.500000
-2022-05-12,FALSE,25.800000
-2022-05-12,TRUE,24.500000
-2022-05-23,FALSE,29.500000
-2022-05-23,TRUE,24.000000
-2022-05-26,FALSE,34.571429
-2022-05-29,TRUE,46.000000
-2022-02-03,FALSE,0.000000
-2022-02-18,TRUE,22.666667
-2022-03-09,TRUE,16.000000
-2022-03-20,TRUE,28.000000
-2022-03-24,FALSE,21.666667
-2022-03-25,FALSE,31.750000
-2022-04-01,TRUE,15.250000
-2022-04-02,FALSE,36.714286
-2022-04-02,TRUE,29.600000
-2022-04-09,FALSE,22.750000
-2022-04-11,FALSE,11.000000
-2022-04-16,TRUE,24.666667
-2022-04-24,TRUE,20.666667
-2022-05-10,FALSE,17.333333
-2022-05-10,TRUE,32.500000
-2022-05-19,FALSE,26.666667
-2022-05-25,TRUE,40.000000
-2022-05-30,TRUE,19.000000
-2022-03-10,FALSE,42.500000
-2022-05-27,TRUE,22.250000
-2022-03-04,TRUE,23.000000
-2022-04-27,TRUE,41.333333
-2022-02-09,TRUE,28.333333
-2022-02-12,TRUE,24.250000
-2022-03-05,TRUE,20.750000
-2022-03-28,TRUE,34.333333
-2022-04-18,FALSE,31.750000
-2022-04-21,FALSE,16.666667
-2022-05-03,TRUE,23.750000
-2022-05-15,TRUE,24.166667
-2022-03-09,FALSE,29.500000
-2022-05-05,FALSE,0.000000
-2022-05-17,TRUE,24.800000
-2022-05-31,FALSE,18.333333
-2022-03-10,TRUE,19.400000
-2022-05-24,FALSE,7.500000
-2022-03-14,FALSE,32.000000
-2022-04-05,TRUE,29.000000
-2022-05-27,FALSE,40.000000
-2022-01-21,FALSE,28.333333
-2022-04-07,TRUE,7.500000
-2022-04-26,TRUE,34.000000
-2022-03-23,FALSE,30.000000
-2022-05-14,TRUE,14.666667
-2022-05-18,FALSE,22.400000
-2022-03-17,TRUE,0.000000
-2022-04-28,FALSE,33.000000
-2022-05-11,TRUE,19.000000
-2022-02-10,TRUE,22.250000
-2022-04-09,TRUE,24.666667
-2022-04-16,FALSE,33.750000
-2022-04-26,FALSE,32.666667
-2022-05-05,TRUE,0.000000
-2022-05-19,TRUE,26.833333
-2022-05-21,FALSE,32.000000
-2022-05-25,FALSE,17.500000
-2022-05-13,FALSE,15.666667
-2022-05-22,TRUE,11.000000
-2022-05-31,TRUE,31.000000
-2022-03-27,FALSE,0.000000
-2022-04-11,TRUE,29.000000
-2022-05-20,FALSE,19.600000
\ No newline at end of file
+﻿date_month,had_discount,base_average_metric
+2022-01-01,TRUE,1.00000000000000000000
+2022-01-01,FALSE,1.00000000000000000000
+2022-02-01,FALSE,1.00000000000000000000
+2022-02-01,TRUE,1.00000000000000000000
```

Comment:
This isn't testing much of anything (as mentioned above)

Reply:
This is fixed in the pytest framework - everything within the integration tests folder is a holdover from when we were doing it that way and I'm hesitant to delete anything until we've got it all migrated over.